test/Microsoft.DotNet.Watcher.Tools.FunctionalTests/Scenario/ProjectToolScenario.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
We needed to change the `version` portion of  `AddToolToProject`. It didn't look like this type was used in any other project so it makes sense to localize it.